Lists/MoreExercises/Messaging/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+
+
+namespace Messaging
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            List<string> list = Console.ReadLine().Split(' ').ToList();
+            List<char> charText = Console.ReadLine().ToList();
```

Comment:
Just use .Split?

Reply:
Do you need charText to be a list a characters? It's basically a string (which is also an array).